apps/cli/src/commands/boot-challenger.ts

```diff
@@ -93,48 +94,64 @@ const initCli = async (commandInstance: Vorpal.CommandInstance) => {
 const onAssertionConfirmedCb = async (nodeNum: any, commandInstance: Vorpal.CommandInstance) => {
     commandInstance.log(`[${new Date().toISOString()}] Assertion confirmed ${nodeNum}. Looking up the assertion information...`);
 
-    if (CHALLENGER_INSTANCE != 1) {
-        commandInstance.log(`[${new Date().toISOString()}] Backup challenger waiting for delay ${(CHALLENGER_INSTANCE) + (BACKUP_SUBMISSION_DELAY / (60 * 1000))} minutes..`);
-        const currentTime = Date.now();
-        await new Promise((resolve) => {
-            setTimeout(resolve, (CHALLENGER_INSTANCE * 60 * 1000) + BACKUP_SUBMISSION_DELAY)
-        });
+    
+    const {isSubmitTime, currentChallenge} = await isChallengeSubmitTime();
+    const lastChallengeTime = Number(currentChallenge.assertionTimestamp);
```

Comment:
This is not the correct field, the assertionTimestamp is the time when the Node got created on the Rollup not when the challenge got submitted to the Referee. You need to use `currentChallenge.createdTimestamp`.

Reply:
As mentioned above, thank you for the distinction, this has been updated.